pkg/utils/selector.go

```diff
@@ -120,8 +120,14 @@ func SelectPods(ctx context.Context, c client.Client, selector v1alpha1.Selector
 	if len(selector.FieldSelectors) > 0 {
 		listOptions.FieldSelector = fields.SelectorFromSet(selector.FieldSelectors)
```

Comment:
Can you please put this line to the line 123? It is not necessary to have two same `if` statement